src/js/components/landing/recentActivity/RecentActivityTable.jsx

```diff
@@ -114,6 +114,21 @@ export default class RecentActivityTable extends React.Component {
         return year+"-"+month+"-"+day;
     }
 
+    hasPermission(agency_name){
```

Comment:
This is identical to the `hasPermission` in DashboardTable.jsx. It should be moved to a helper to be reused instead of duplicated in multiple places